_posts/minutes/2016-10-07-minutes.markdown

```diff
@@ -1,4 +1,4 @@
-[LISTEN TO THIS MEETING](https://fccdl.in/iPvoj2r1R)
+&nbsp;&nbsp;&nbsp;Listen to this meeting here: [here](https://fccdl.in/iPvoj2r1R)

```

Comment:
Hey @pathawks, did everything look okay with this approach? I just picked something simple so we could get these meetings minutes up.